pvlib/tests/test_inverter.py

```diff
@@ -95,6 +95,45 @@ def test_sandia_Pnt_micro():
     assert_series_equal(pacs, pd.Series([-0.043, 132.545914746, 240.0]))
 
 
+def test_sandia_multi(cec_inverter_parameters):
+    vdcs = pd.Series(np.linspace(0, 50, 3))
+    idcs = pd.Series(np.linspace(0, 11, 3)) / 2
+    pdcs = idcs * vdcs
+    pacs = inverter.sandia_multi((vdcs, vdcs), (pdcs, pdcs),
+                                 cec_inverter_parameters)
+    assert_series_equal(pacs, pd.Series([-0.020000, 132.004308, 250.000000]))
+
+
+def test_sandia_multi_error(cec_inverter_parameters):
+    vdcs = pd.Series(np.linspace(0, 50, 3))
+    idcs = pd.Series(np.linspace(0, 11, 3)) / 2
+    pdcs = idcs * vdcs
+    with pytest.raises(ValueError, match='p_dc and v_dc have different'):
+        inverter.sandia_multi((vdcs,), (pdcs, pdcs), cec_inverter_parameters)
+
+
+def test_sandia_multi_array(cec_inverter_parameters):
+    vdcs = np.linspace(0, 50, 3)
+    idcs = np.linspace(0, 11, 3)
+    pdcs = idcs * vdcs
+    pacs = inverter.sandia_multi(vdcs, pdcs, cec_inverter_parameters)
+    assert_allclose(pacs, np.array([-0.020000, 132.004278, 250.000000]))
+
+
+def test_sandia_multi_float(cec_inverter_parameters):
```

Comment:
@wholmgren do you think the use case in this test should be retained?

Reply:
No

Reply:
That will simplify things.